blockchain/standalone/subsidy.go

```diff
@@ -47,7 +47,8 @@ type SubsidyParams interface {
 	// block subsidy between PoW, PoS, and the Treasury.
```

Comment:
Copyright year can be bumped to `2021` for this file and in `blockchain/standalone/subsidy_test.go` as well.

Reply:
Good call.   Updated both.